framework/src/main/java/org/checkerframework/framework/stub/RemoveAnnotationsForInference.java

```diff
@@ -51,6 +56,12 @@
  *
  * <p>Does not remove trusted annotations: those that the checker trusts rather than verifies.
  *
+ * <p>Also does not remove "excluded" annotations. Provide excluded annotations with the {@code
+ * -exclusionFile} command line argument, which must be the first argument to this program if it is
+ * present. The exclusion file should be a list of newline-separated annotation names (without
+ * {@literal @} symbols). Both the simple and fully-qualified name of each annotation must be
+ * included in the exclusion file. TODO: remove this restriction?
```

Comment:
"must be included": presumably only if both forms appear in the source code and neither should be removed.